packages/client/src/generation/TSClient/TSClient.ts

```diff
@@ -96,7 +96,7 @@ export class TSClient implements Generatable {
     const relativeOutdir = path.relative(process.cwd(), outputDir)
 
     const code = `${commonCodeJS({ ...this.options, browser: false })}
-${buildRequirePath(dataProxy)}
+${buildRequirePath(edge)}
```

Comment:
On Node.js we need to import `path` because we do env var loading/checking in `buildWarnEnvConflicts`, which uses `path`.